weeklymission2/src/Components/FolderHeader.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import logo from "../images/logo.svg";
+import linkIcon from "../images/link.svg";
+
+export function FolderHeader({ profile }) {
+  return (
+    <>
+      <div className="headerArea">
+        <nav>
+          <div className="navBar">
```

Comment:
`div`를 `nav`로 대체해도 됩니다. 마크업은 최대한 간결하게 ! 
div와 같은 블록요소로서 같은 역할을 합니다. 

```suggestion
        <nav class="navBar">
```